CMakeLists.txt

```diff
@@ -1830,7 +1830,12 @@ MYSQL_CHECK_LDAP()
 MYSQL_CHECK_LDAP_DLLS()
 
 # Percona LDAP Simple / LDAP SASL authentication plugins
-OPTION(WITH_PERCONA_AUTHENTICATION_LDAP "Build with Percona LDAP Simple / LDAP SASL authentication plugins" ON)
+# Problem with  MacOS system libraries and complains for compilation error ldap_set_urllist_proc
```

Comment:
@rahulmalik87 just a stylistic comment. From what I see from Oracle's CMake files, they usually do this as following
```
IF(APPLE)
  SET(WITH_PERCONA_AUTHENTICATION_LDAP_DEFAULT OFF)
ELSE()
  SET(WITH_PERCONA_AUTHENTICATION_LDAP_DEFAULT ON)
ENDIF()
OPTION(WITH_PERCONA_AUTHENTICATION_LDAP "Build with Percona LDAP Simple / LDAP SASL authentication plugins" ${WITH_PERCONA_AUTHENTICATION_LDAP_DEFAULT})
```

Reply:
fixed